corpus/templates/base.html

```diff
@@ -43,20 +43,42 @@
 
 <div class="navbar bg-primary text-primary-content sticky top-0 z-10">
     <div class="navbar-start">
+        <a href="{% url 'index' %}" class="btn btn-ghost normal-case text-xl">Corpus</a>
+    </div>
+    <div class="navbar-center">
+        <a href="{% url 'index' %}" class="btn btn-ghost normal-case text-xl">Home</a>
+        <a href="{% url 'about_us' %}" class="btn btn-ghost normal-case text-xl">About Us</a>
         <div class="dropdown">
-            <label tabindex="0" class="btn btn-ghost btn-circle">
-                <svg xmlns="http://www.w3.org/2000/svg" class="h-5 w-5" fill="none" viewBox="0 0 24 24"
-                     stroke="currentColor">
-                    <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h7"/>
-                </svg>
+            <label tabindex="0" class="btn btn-ghost normal-case text-xl">
+                SIGs
             </label>
-            {% include "components/navbar_dropdown.html" %}
+            {% include "components/general_dropdown_sigs.html" %}
         </div>
-    </div>
-    <div class="navbar-center">
-        <a href="{% url 'index' %}" class="btn btn-ghost normal-case text-xl">Corpus</a>
+        <div class="dropdown">
+            <label tabindex="0" class="btn btn-ghost normal-case text-xl">
+                Affinity Groups
+            </label>
+            {% include "components/general_dropdown_affinity_groups.html" %}
+        </div>
+        <div class="dropdown">
+            <label tabindex="0" class="btn btn-ghost normal-case text-xl">
+                Embedathon
+            </label>
+            {% include "components/general_dropdown_embedathon.html" %}
+        </div>
+        <a href="#" class="btn btn-ghost normal-case text-xl">Blog</a>
+        <a href="#" class="btn btn-ghost normal-case text-xl">Gyan</a>
+        <a href="#" class="btn btn-ghost normal-case text-xl">Achievements</a>
+        <a href="#" class="btn btn-ghost normal-case text-xl">Virtual Expo</a>
```

Comment:
```suggestion
            <label tabindex="0" class="btn btn-ghost normal-case text-lg">
                Embedathon
            </label>
            {% include "components/general_dropdown_embedathon.html" %}
        </div>
        <a href="https://ieee-nitk.github.io/blog/" class="btn btn-ghost normal-case text-lg">Blog</a>
        <a href="https://ieee-nitk.github.io/gyan/" class="btn btn-ghost normal-case text-lg">Gyan</a>
        <a href="https://ieee-nitk.github.io/virtual-expo/" class="btn btn-ghost normal-case text-lg">Virtual Expo</a>
```

Removed the Achievements section link until we create the Achievements section in Corpus. Tracked in #55

Reply:
Alright, resolved this too